Directory.Build.props

```diff
@@ -14,6 +14,7 @@
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
     <EnableWindowsTargeting>true</EnableWindowsTargeting>
     <TargetingWindows Condition=" '$(TargetFramework)' == 'net472' or $(TargetFramework.EndsWith('-windows')) ">true</TargetingWindows>
+    <TargetingMacos Condition=" '$(TargetFramework)' == 'net6.0-macos10.15' or $(TargetFramework.EndsWith('-macos')) ">true</TargetingMacos>
```

Comment:
There is a contains syntax that may be better to use here if you're liable to use `-macos10.15`, so that the condition doesn't silently start failing when you later move to `-macos12`.
I think it's roughly this:
` $(TargetFramework.Contains('macos')) `